scripts/build.tokens.ckerc20.mjs

```diff
@@ -78,6 +79,37 @@ const ORCHESTRATOR_PRODUCTION_ID = Principal.fromText('vxkom-oyaaa-aaaar-qafda-c
 
 const DATA_FOLDER = join(process.cwd(), 'src', 'frontend', 'src', 'env');
 
+const LOGO_FOLDER = join(process.cwd(), 'src', 'frontend', 'src', 'icp-eth', 'assets');
+
+const saveTokenLogo = async (canisterId, name) => {
+	const logoName = name.toLowerCase().replace('ck', '').replace('sepolia', '');
+	const file = join(LOGO_FOLDER, `${logoName}.svg`);
+
+	if (existsSync(file)) {
+		return;
+	}
+
+	const agent = await createAgent({
```

Comment:
should we move the `agent` level up? that way the same variable can be used in `saveTokenLogo` and `orchestratorInfo`?

Reply:
yes, makes sense!